app/models/item.rb

```diff
@@ -5,13 +5,11 @@ class Item <ApplicationRecord
   has_many :orders, through: :item_orders
   has_many :users, through: :merchant
 
-  validates_presence_of :name,
-                        :description,
-                        :price,
-                        :image,
-                        :inventory
+  validates_presence_of :name, :description, :price, :inventory
+  validates :image, presence: true, allow_blank: true
   validates_inclusion_of :active?, :in => [true, false]
   validates_numericality_of :price, greater_than: 0
+  validates_numericality_of :inventory, greater_than_or_equal_to: 0
```

Comment:
Should we add the only_integer validation for inventory as well?

Reply:
Ooh I like that idea! That was one of the things I had written down as feedback from Little Shop